ansible/roles/artifacts/tasks/main.yaml

```diff
@@ -530,6 +530,45 @@
     mode: "644"
     checksum: sha256:215493daac110055990318b7916fbf1a7d6b08feea9c27cdd6403df31557d992
 
+# tensorrt_vad
+- name: Create tensorrt_vad directory inside {{ data_dir }}
+  ansible.builtin.file:
+    path: "{{ data_dir }}/vad"
+    mode: "755"
+    state: directory
+
+- name: Download tensorrt_vad/vad-carla-tiny_backbone.onnx
+  become: true
+  ansible.builtin.get_url:
+    url: https://awf.ml.dev.web.auto/planning/models/tensorrt_vad/carla_tiny/v0.1/vad-carla-tiny_backbone.onnx
+    dest: "{{ data_dir }}/vad/vad-carla-tiny_backbone.onnx"
```

Comment:
```suggestion
    dest: "{{ data_dir }}/vad/v1.0/vad-carla-tiny_backbone.onnx"
```
If you plan to update versions in the future, you need to include version numbers in the directory name; otherwise, Ansible won’t be able to perform the update.